scorecards-site/static/viewer/index.html

```diff
@@ -850,6 +860,15 @@
                   </a>
                   </div>
               </div>
+              <div class="content"><small class="fw-semibold">API URL:</small>
+                {{#if apiURL}}
+                  <a class="btn-muted-link" href="{{apiURL}}" target="_blank" rel="noopener noreferrer">
+                    https://api.scorecard.dev/{{data.repo.name}}
```

Comment:
This link (even though the `href` works) should be:
```
https://api.scorecard.dev/projects/{{data.repo.name}}
```

Reply:
Note: this doesn't have the commit part either, but I think thats ok

Reply:
added `/projects/` but leaving the commit out for readability purposes.